compiler/rustc_passes/src/check_attr.rs

```diff
@@ -597,13 +604,24 @@ impl CheckAttrVisitor<'tcx> {
                                     .emit();
                                 },
                             );
-                            return false;
+                            is_valid = false;
                         }
                     }
+                } else {
+                    self.tcx.struct_span_lint_hir(
+                        INVALID_DOC_ATTRIBUTES,
+                        hir_id,
+                        meta.span(),
+                        |lint| {
+                            lint.build(&format!("unknown `doc` attribute")).emit();
```

Comment:
Maybe this should say ``invalid `doc` attribute`` instead, since it's not really "unknown", it's rather "invalid".

Reply:
I think it should say ``invalid `doc` attribute`` too.

Reply:
Done.